### PR TITLE
DM-2674: Fixed bug with changing the microservice

### PR DESCRIPTION
**Pull request by shaikhahme**

### CODE REVIEW

File: content/change-logs/device-management/cumulocity-10-20-316-0-Fixed-issue-with-changing-microservice-service-key.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    label: LWM2M
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: DM-2674
+version: 10.20.317.0
+---
+
+Changing the external microservice was not working as expected because the LWM2M agent would still point the custom
+actions to the old microservice as the service-key was not being updated.This LWM2M agent version fixes the bug by
```
Comment (BeateRixen) [inline content/change-logs/device-management/cumulocity-10-20-316-0-Fixed-issue-with-changing-microservice-service-key.md:19]:
```suggestion
actions to the old microservice as the servicekey was not being updated. Now 
```

File: content/change-logs/device-management/cumulocity-10-20-316-0-Fixed-issue-with-changing-microservice-service-key.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+updating the service-key when the external microservice is changed.
```
Comment (BeateRixen) [inline content/change-logs/device-management/cumulocity-10-20-316-0-Fixed-issue-with-changing-microservice-service-key.md:20]:
```suggestion
the service key is updated properly when the external microservice is changed.
```

File: content/change-logs/device-management/cumulocity-10-20-316-0-Fixed-issue-with-changing-microservice-service-key.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  - value: component-1KLUzmqfe
+    label: LWM2M
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: DM-2674
+version: 10.20.317.0
+---
+
+In LWM2M device protocols resource's custom action configuration, the exchange to another external decoder microservice
```
Comment (BeateRixen) [inline content/change-logs/device-management/cumulocity-10-20-316-0-Fixed-issue-with-changing-microservice-service-key.md:18]:
```suggestion
In the custom action configuration of LWM2M device protocol resources, the exchange to another external decoder microservice
```

**Review by BeateRixen — APPROVED** [review]

File: content/change-logs/device-management/cumulocity-10-20-316-0-Fixed-issue-with-changing-microservice-service-key.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    label: Fix
+product_area: Device Management & Connectivity
+component:
+  - value: component-1KLUzmqfe
+    label: LWM2M
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: DM-2674
+version: 10.20.317.0
```
Comment (Oezge-Akin) [inline content/change-logs/device-management/cumulocity-10-20-316-0-Fixed-issue-with-changing-microservice-service-key.md:15]:
```suggestion
version: 10.20.319.0
```
according to the commit history:
https://github.softwareag.com/IOTA/c8y-lwm2m/commits/develop

File: content/change-logs/device-management/cumulocity-10-20-316-0-Fixed-issue-with-changing-microservice-service-key.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+change_type:
+  - value: change-VSkj2iV9m
+    label: Fix
+product_area: Device Management & Connectivity
+component:
+  - value: component-1KLUzmqfe
+    label: LWM2M
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
```
Comment (Oezge-Akin) [inline content/change-logs/device-management/cumulocity-10-20-316-0-Fixed-issue-with-changing-microservice-service-key.md:13]:
```suggestion
  - value: tc-ggH2M4hf3
    label: lwm2m-agent
```
now we have lwm2m-agent component. This one is relevant for changes created from c8y-lwm2m component based repo.
https://sagportal.sharepoint.com/:w:/s/IoTAnalyticsRnDOps/ERa81HCQMrlBpRXs9W_LuVgBHESu5E6aqoRLUrGD3UGI-w?e=AhgG9Z

**Review by Oezge-Akin — COMMENTED** [review]
Could you also change the filename to 
lwm2m-agent-10-20-319-0-...

**Review by Oezge-Akin — APPROVED** [review]

**Comment by BeateRixen** [comment]
@shaikhahme Can we merge this?

**Pull request merged**